### PR TITLE
Update homepage card section headings and add program count

### LINKED ISSUE #8246: [Program Filtering] Update section labels on the homepage and add program counts
#### Describe the solution presented in this issue.
Based on the mocks for the Program Filtering epic, the wording for section titles on the homepage should be updated like this:

- The pre-screener section should be titled “Find services” rather than “Get started”
- If there’s a pre-screener, the unapplied programs section will be titled “Programs and services” rather than “All programs”
- If there’s a “Recommended“ section, the other section will be titled “Other programs and services” rather than “Not started”

Each section (other than the pre-screener) should also have a count next to the title, like this:

image

#### Link to Figma mocks: 
[mocks](https://www.figma.com/file/BOES7eVIpEFIbBBShCN8AB/Program-card-discoverability?type=design&node-id=239-11037&mode=design&t=fzLPhaaMXNqOQ6yg-0)

#### Done when:

- [ ] The title of each section on the homepage matches the mocks
- [ ] Each section has a count in parentheses next to the title

### PR DESCRIPTION
**Pull request by melanie-exygy**
### Description

Based on the mocks for the Program Filtering epic, update the wording for section headings on the homepage like this:

- The pre-screener section should be titled “Find services” rather than “Get started”
- If there’s a pre-screener, the unapplied programs section will be titled “Programs and services” rather than “All programs”
- If there’s a “Recommended“ section, the other section will be titled “Other programs and services” rather than “Not started”

Also add a program card count in each heading (other than the pre-screener).

### Issue(s) this completes

Fixes #8246

### CODE REVIEW

File: server/conf/i18n/messages [hunk 1]
```diff
@@ -142,12 +142,18 @@ title.inProgressProgramsUpdated=In progress
 title.activeProgramsUpdated=Not started
 # Title of the section on the home page with the benefits finder form.
 title.benefitsFinderSection=Find services that can help you
+# Title of the section on the home page with the benefits finder form.
+title.benefitsFinderSection.v2=Find services
 # Title of the section on the home page that shows programs for which the applicant has already started or submitted applications.
 title.myApplicationsSection=My applications ({0})
 # Title of the section on the home page that shows any programs that don't match the selected filters.
 title.otherProgramsSection=Other programs ({0})
+# Title of the section on the home page that shows any programs that don't match the selected filters.
```
Comment (dkatzz) [inline server/conf/i18n/messages:151]: **suggestion:** Maybe add "with the number of programs in parentheses" - here and below - to give some extra context.

**Review by dkatzz — APPROVED** [review]

File: server/conf/i18n/messages (hunk 1 above)
Comment (melanie-exygy) [reply server/conf/i18n/messages:151]: I really like that idea.  Do you know if it's okay for me to update the comment for messages that have already been translated?  Will that mess up anything in Transifex?

Comment (melanie-exygy) [reply server/conf/i18n/messages]: Done.  I only updated the comments for strings that I was adding in this PR, just to make sure I didn't mess anything up with Transifex.

Comment (dkatzz) [reply server/conf/i18n/messages]: Should be fine either way, but sounds good!! I'm up next to do translations so I can update those before I send them out. Thanks!

**Pull request merged**